modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -375,10 +353,11 @@ async fn search_advisories(ctx: TrustifyContext) -> Result<(), anyhow::Error> {
     assert_eq!(result.total, 0);
 
     // ingest some advisories
-    let data = include_bytes!("../../../../../etc/test-data/mitre/CVE-2024-27088.json");
-    let _id = ingest(&ingestor, data).await;
-    let data = include_bytes!("../../../../../etc/test-data/mitre/CVE-2024-28111.json");
-    let _id = ingest(&ingestor, data).await;
+    ctx.ingest_documents([
+        (Format::CVE, "mitre/CVE-2024-27088.json"),
+        (Format::CVE, "mitre/CVE-2024-28111.json"),
+    ])
+    .await?;
```

Comment:
Why pass the `Format`? Kinda defeats the whole purpose of `Format::from_bytes`, which is the code path that should be tested, imo.

Reply:
Ingestor's ingest() takes a Format. We should make a variant that doesn't require it. try_ingest() maybe.